apollo-api/src/commonMain/kotlin/com/apollographql/apollo3/api/ApolloExperimental.kt

```diff
@@ -0,0 +1,10 @@
+package com.apollographql.apollo3.api
+
+@RequiresOptIn(
+    level = RequiresOptIn.Level.WARNING,
+    message = "This API is experimental and subject to change, use with caution"
```

Comment:
Nit: "use with caution" might scare legitimate users? I'd keep it factual:

```suggestion
    message = "This API is experimental and can be changed in a backwards-incompatible manner."
```